### ISSUE TITLE
Why there exists so many difference between provided training log and config file of hybrik?

### ISSUE THREAD

**Issue by YongtaoGe** [open-mmlab/mmhuman3d#42]
https://openmmlab-share.oss-cn-hangzhou.aliyuncs.com/mmhuman3d/models/hybrik/20211109_164017.log?versionId=CAEQHhiBgICdvonS6xciIDdiNGYzY2Q3N2NiMTQ5MzdhOTZjYjEwZDM0ZjI3ODU1

**Comment by YongtaoGe** [comment]
I failed to reproduce the result of hybrik. Then I found the loss weight and augmentation hyperparameters and many configurations are inconsistent.

**Comment by pangyyyyy** [comment]
Hi @YongtaoGe We experimented with tuning the hyperparameters i.e.increasing twist loss and augmentations and found that these changes could achieve a better result. Thanks for pointing out the discrepancy between the provided logs and config file, we will update our config in the future.

**Comment by YongtaoGe** [comment]
@pangyyyyy the provided "hybrik_h36m_train.npz" does not contain smpl parameters. If tried to convert it from the script, however, the code will report memory error when "num_workers_per_gpu" is set larger than 0.

**Comment by pangyyyyy** [comment]
@YongtaoGe, running this command should give you the preprocessed  `hybrik_h36m_train.npz`:
```
python tools/convert_datasets.py \
  --datasets h36m_hybrik \
  --root_path data/datasets \
  --output_path data/preprocessed_datasets
```

I think there isn't an option to set the `num_workers_per_gpu` for dataset conversion. For the memory error issue, are you referring to training with the preprocessed npz file?

**Comment by YongtaoGe** [comment]
@pangyyyyy Yes, I refer to training with the preprocessed npz file.

**Comment by pangyyyyy** [comment]
@YongtaoGe Do you mind providing me with your system specs? I'll try to see if i can replicate this error

**Comment by YongtaoGe** [comment]
@pangyyyyy My enviroment looks like:
```
sys.platform: linux
Python: 3.6.13 | packaged by conda-forge | (default, Feb 19 2021, 05:36:01) [GCC 9.3.0]
CUDA available: True
GPU 0: GRID T4-8C
CUDA_HOME: /usr/local/cuda-11.0
NVCC: Build cuda_11.0_bu.TC445_37.28540450_0
GCC: gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-44)
PyTorch: 1.7.1+cu110
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191122 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v1.6.0 (Git Hash 5ef631a030a6f73131c77892041042805a06064f)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.0
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80
  - CuDNN 8.0.5
  - Magma 2.5.2
  - Build settings: BLAS=MKL, BUILD_TYPE=Release, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DUSE_VULKAN_WRAPPER -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, USE_CUDA=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, 

TorchVision: 0.8.2+cu110
OpenCV: 4.5.4
MMCV: 1.3.15
MMCV Compiler: GCC 8.3
MMCV CUDA Compiler: 11.0
MMHuman3d: 0.3.0+
```

**Comment by pangyyyyy** [comment]
@YongtaoGe The error you posted looks more like a cpu issue. What's your cpu count/specs and how much RAM memory do you have?

**Comment by YongtaoGe** [comment]
@pangyyyyy 
I think I have enough RAM meomory and cpu numbers
'''
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                96
On-line CPU(s) list:   0-95
Thread(s) per core:    2
Core(s) per socket:    24
Socket(s):             2
NUMA node(s):          2
Vendor ID:             GenuineIntel
CPU family:            6
Model:                 85
Model name:            Intel(R) Xeon(R) Platinum 8255C CPU @ 2.50GHz
Stepping:              7
CPU MHz:               2501.000
CPU max MHz:           2501.0000
CPU min MHz:           1000.0000
BogoMIPS:              5001.46
Virtualization:        VT-x
L1d cache:             32K
L1i cache:             32K
L2 cache:              1024K
L3 cache:              36608K
NUMA node0 CPU(s):     0-23,48-71
NUMA node1 CPU(s):     24-47,72-95
'''
